frontend/packages/citizen-frontend/src/messages/MessageListItem.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+
+import styled from 'styled-components'
+import colors from '@evaka/lib-components/src/colors'
+import { defaultMargins } from '@evaka/lib-components/src/white-space'
+import { ReceivedBulletin } from '~messages/types'
+import { formatDate } from '~util'
+import { FixedSpaceColumn } from '@evaka/lib-components/src/layout/flex-helpers'
+import LocalDate from '@evaka/lib-common/src/local-date'
+
+type Props = {
+  bulletin: ReceivedBulletin
+  active: boolean
+  onClick: () => void
+}
+function MessageListItem({ bulletin, active, onClick }: Props) {
+  return (
+    <Container
+      isRead={bulletin.isRead}
+      active={active}
+      onClick={onClick}
+      tabIndex={0}
+      onKeyPress={(e) => e.key === 'Enter' && onClick()}
```

Comment:
I don't know if just using a `button` instead of manually setting these whould be a better idea in terms of accessibility. Seems kind of weird in a way, though.

Reply:
Hmm, good point.. button seems also kinda weird as a list row item though.. @massproduction got any wisdom on this?

Reply:
Yeah if it behaves like a button then I would just use a button.

Reply:
changed to button, will consider once more when we are closer to getting to prod